bookfinder/src/components/Footer.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { FaFacebook, FaInstagram, FaGithub } from "react-icons/fa";
+
+const Footer = () => {
+  return (
+    <div className="top-64 flex h-full w-full flex-col gap-10 bg-gray-300 p-10 xl:flex-row xl:gap-56">
+      <div className="flex flex-col gap-5">
+        <div className="text-xl font-semibold">Follow Us</div>
```

Comment:
and here `md:flex-row`. It worked by mine. You can try it also for **categories** and **contact us** part

Reply:
I tried `md:flex-row` to start with, but the content was getting cropped on some `lg` screens, but using `xl:flex-row` handled that. Behaves the same way like `flex-wrap` now